compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/lower_to_ukernel.mlir

```diff
@@ -1,5 +1,3 @@
-// RUN: iree-opt --split-input-file --pass-pipeline="builtin.module(func.func(iree-amdaie-lower-to-ukernels,cse,canonicalize))" %s | FileCheck %s
```

Comment:
I still see `CHECK` statements - you may convert them to `PAD_PACK`.

Reply:
All use CHECK statements now